app/assets/javascripts/discourse/widgets/post-small-action.js.es6

```diff
@@ -102,7 +102,7 @@ export default createWidget("post-small-action", {
     contents.push(
       avatarFor.call(this, "small", {
         template: attrs.avatar_template,
-        username: attrs.avatar,
```

Comment:
Wow, that's a weird bug. How long has that been there?

Reply:
Don't you remember doing it 3 years ago ?

<img width="959" alt="screenshot 2019-01-24 at 23 01 29" src="https://user-images.githubusercontent.com/339945/51711324-08374300-202c-11e9-980d-5635dcaa586d.png">

:trollface:

Reply:
In fact it was probably working when you did it maybe ? maybe `attrs.avatar` was different at that time ?

Reply:
Totally possible.